index.html

```diff
@@ -27,7 +27,7 @@ <h1>
           <li><a href="https://github.com/rusucosmin" target="_blank">Cosmin Rusu</a></li>
           <li><a href="https://github.com/RusuRobert" target="_blank">Robert Rusu</a></li>
           <li><a href="https://github.com/RNarcis" target="_blank">Narcis Rusu</a></li>
-          <li><a href="https://github.com/RusuAlexPaul" target="_blank">Alex Paul Rusu</a></li>
+          <li><a href="https://github.com/RusuAlexPaul" target="_blank">Alex P. Rusu</a></li>
```

Comment:
Thanks Paul! Can you actually just remove P. For consistency? Both Robert, Narcis and me have just our first and last name.